frontends/bmd/src/pages/poll_worker_screen.tsx

```diff
@@ -173,9 +164,15 @@ export function PollWorkerScreen({
       ? 'All Precincts'
       : find(election.precincts, (p) => p.id === appPrecinct.precinctId).name;
 
-  const precinctBallotStyles = cardlessVoterSessionPrecinctId
+  const [selectedCardlessVoterPrecinctId, setSelectedCardlessVoterPrecinctId] =
```

Comment:
There's an intermediate state when a pollworker is activating a cardless voter session where they have selected a precinct but have yet to select a ballot style. Previously, that was represented in the main app state by having the user precinct id defined and the ballot style id undefined.

Since that state got moved to the `activatedCardlessVoter` field in the pollworker auth, I thought it felt a little confusing to have a partially activated cardless voter. Instead, I added local component state for the selected precinct id during the activation process.